### PR TITLE
(maint) the required docker changes for puppetserver work

### PR DESCRIPTION
**Pull request by Thomas-Franklin**

### CODE REVIEW

File: spec/Dockerfile.puppetdb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+FROM puppet/puppetdb
```
Comment (da-ar) [inline spec/Dockerfile.puppetdb]: what is puppetdb needed for in our workflow?

File: spec/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,25 @@
+# Travis not using 18.04 yet
```
Comment (da-ar) [inline spec/Dockerfile]: what does this give us?

File: spec/Dockerfile.puppetdb (hunk 1 above)
Comment (Thomas-Franklin) [reply spec/Dockerfile.puppetdb]: I thought this is where the puppetserver gets the data about tasks from, I may be wrong

File: spec/Dockerfile (hunk 2 above)
Comment (Thomas-Franklin) [reply spec/Dockerfile]: Plain copypasta from bolt, as these dockerfiles have not been changed

Comment (Thomas-Franklin) [reply spec/Dockerfile]: Good point highlighting this, forgot to remove this dockerfile

**Comment by da-ar** [comment]
https://github.com/puppetlabs/ace/pull/18 supersedes this PR

**Pull request merged**